packages/gatsby/src/utils/html-renderer-queue.js

```diff
@@ -22,7 +22,15 @@ module.exports = (htmlComponentRendererPath, pages, activity) =>
       pageSegment =>
         new Promise((resolve, reject) => {
           workerPool
-            .renderHTML({ htmlComponentRendererPath, paths: pageSegment })
+            .renderHTML({
+              htmlComponentRendererPath,
+              paths: pageSegment,
+              envVars: {
```

Comment:
Add a comment mentioning these are just variables that we set internally and we don't need to set others as the child process will pick them up from the environment.

Reply:
Done